pym/portage/dbapi/vartree.py

```diff
@@ -3522,6 +3552,8 @@ def _collision_protect(self, srcroot, destroot, mypkglist,
 						if fnmatch.fnmatch(f_match, pattern):
 							stopmerge = False
 							break
+					if self._is_install_masked(relative_path, install_mask):
+						stopmerge = False
```

Comment:
This case will never trigger because treewalk has already filtered any files matched by install_mask. If install-masked files are included in CONTENTS, then the _collision_protect isowner calls can return incorrect results:
```python
				for ver in mypkglist:
					if ver.isowner(f):
						isowned = True
						break
```
Also, this code in treewalk can return incorrect results:
```python
					owners = self.vartree.dbapi._owners.get_owners(collisions)
```